site/content/contribute/more-info/webapp/build-component.md

```diff
@@ -21,8 +21,6 @@ If none of those make any sense to you or you're new to React and Redux, then ch
 - https://facebook.github.io/react/tutorial/tutorial.html
 - http://redux.js.org/
 
-The components for the webapp currently live in the `components/` directory in the {{< newtabref href="https://github.com/mattermost/mattermost-webapp" title="mattermost-webapp repository" >}}.
```

Comment:
These live in multiple places now, and since I couldn't explain that in a concise way, I just removed this sentence